lib/sinon/spy.js

```diff
@@ -395,6 +395,12 @@ function delegateToCalls(method, matchAny, actual, notCalled) {
             return false;
         }
 
+        if (totalCallCount !== undefined) {
```

Comment:
Try to avoid unnecessary indentation when there's only one possible outcome

```js
if (totalCallCount !== undefined && this.callCount !== totalCallCount) {
    return false;
 }
```